### DISCUSSION TITLE
How do I group node/npm/pnpm/yarn version updates?

### DISCUSSION THREAD

**Discussion by KnifeFed** [renovatebot/renovate#21537] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I currently have two pull requests from Renovate:

1. `update node.js to v18.16.0`
    - Updates `.node-version`, `.nvmrc`, `.tool-versions`, `engines` in `package.json`, `pnpm-lock.yaml`.
2. `update pnpm to v8.2.0`
    - Updates `.tool-versions`, `engines` + `packageManager` in `package.json`, `pnpm-lock.yaml`.
 
How do I group updates like this?

### What you tried so far.

Nothing. I've yet to find the relevant info in the docs.

**Comment by viceice** [comment, marked as answer]
https://docs.renovatebot.com/noise-reduction/#package-grouping

**Reply by KnifeFed** [reply]
Ah, I didn't see these as "packages" as I thought that just referred to e.g. npm packages. This works well:

```json
  "packageRules": [
    {
      "groupName": "tool versions",
      "matchPackagePatterns": ["node", "npm", "pnpm"]
    }
  ]
```

Thank you.
